tests/ui/cbmc_checks/signed-overflow/expected

```diff
@@ -1,14 +1,8 @@
-Failed Checks: arithmetic overflow on signed addition
-Failed Checks: arithmetic overflow on signed subtraction
-Failed Checks: arithmetic overflow on signed multiplication
 Failed Checks: division by zero
```

Comment:
This is pointed out in the call-outs: specifying any additional checks to the CBMC command (e.g. `--signed-overflow-check`) may become a no-op since the slicer potentially removes code checked by that particular check.